content/chainguard/factory/faq.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Chainguard Factory FAQs"
+linktitle: "FAQs"
+type: "article"
+description: "Chainguard Factory FAQs"
+lead: "An introduction to Chainguard’s Factory; the powerhouse of engineering and automation that continuously transforms the chaos of open source into secure, up-to-date containers, libraries, and VMs at massive scale."
+date: 2025-07-17T08:49:31+00:00
+lastmod: 2025-07-17T08:49:31+00:00
+draft: false
+tags: ["Chainguard Containers", "Chainguard VMs"]
+images: []
+menu:
+  docs:
+    parent: "chainguard-factory"
+weight: 010
+toc: true
+---
+
+## What is the Chainguard Factory?
+
+The Chainguard Factory refers to all the engineering and automation work that goes into building, publishing, and maintaining the software packaged in Chainguard's products. This includes continuously monitoring, testing, and updating thousands of open source projects that make up Chainguard containers, libraries, and VMs.
+
+## How does Chainguard keep its software up to date?
+
+Chainguard uses automated systems to vigilantly monitor for new releases using the GitHub API and release-monitoring.org. When a new release is detected, automation opens a pull request, patches the relevant software, and kicks off a suite of tests. If issues arise, AI analyzes logs and suggests fixes before human engineers review and approve changes. This process ensures prompt, high-quality updates.
```

Comment:
In the other doc, we refer to release-monitoring.org as such:

`the [Release Monitoring project](https://release-monitoring.org/)`

Do we want to do the same here?